docs/resources/shared_database.md

```diff
@@ -0,0 +1,56 @@
+---
+page_title: "snowflake_shared_database Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  A shared database creates a database from a share provided by another Snowflake account. For more information about shares, see Introduction to Secure Data Sharing https://docs.snowflake.com/en/user-guide/data-sharing-intro.
+---
+
+# snowflake_shared_database (Resource)
+
+A shared database creates a database from a share provided by another Snowflake account. For more information about shares, see [Introduction to Secure Data Sharing](https://docs.snowflake.com/en/user-guide/data-sharing-intro).
+
+## Example Usage
+
+```terraform
+resource "snowflake_shared_database" "test" {
+  name                  = "shared_database"
+  from_share            = "organization_name.account_name.share_name"
+  is_transient          = false
+  external_volume       = "external_volume_name"
+  catalog               = "catalog_name"
+  default_ddl_collation = "en_US"
+  log_level             = "OFF"
+  trace_level           = "OFF"
+  comment               = "A shared database"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `from_share` (String) A fully qualified path to a share from which the database will be created. A fully qualified path follows the format of `"<organization_name>"."<account_name>"."<share_name>"`.
```

Comment:
In docs the format is <provider_account>.<share_name>

Reply:
Changed